front/src/routes/integration/all/zigbee2mqtt/setup-page/actions.js

```diff
@@ -38,6 +39,16 @@ const createActions = store => {
           networkModeValid: zigbee2mqttStatus.networkModeValid
         });
       }
+
+      // Compute Zigbee2Mqtt interface URL if connected locally (not through Gladys Plus Gateway)
+      let z2mUrl = null;
+      if (this.session.gatewayClient === undefined) {
```

Comment:
I don't think it's working! this.session doesn't contain the session object, does it?

Shouldn't it be `state.session`, like we do line 27 of this file to get the state.httpClient ?

Reply:
Fixed